executor/runtime/docker/capabilities_linux.go

```diff
@@ -65,9 +83,22 @@ func setupAdditionalCapabilities(c runtimeTypes.Container, hostCfg *container.Ho
 		seccompProfile = "fuse-container.json"
 	}
 
+	// the image building apparmor+seccomp profiles are a strict superset
+	// of the fuse ones, but fuse allows access to /dev/fuse, so these need
+	// to come after fuse in the event that people asked for both
```

Comment:
Its only a strict subset *now*.

This is a new API. Let's start strict.
Please error out when someone asks for both.